.docforge/manifest.yaml

```diff
@@ -1,49 +1,38 @@
-{{- $vers := Split .versions "," -}}
-{{ $defaultBranch := (index $vers 0) }}
 structure:
 - name: _index.md
-  source: https://github.com/gardener/dashboard/blob/{{$defaultBranch}}/README.md
+  source: /README.md
```

Comment:
It seems that the step `concourse-ci/check-docforge` cannot handle this short relative urls.

Reply:
Recently we have updated docforge that now handles relative paths